op-mode-definitions/show-hardware.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="hardware">
+        <properties>
+          <help>Show system hardware details</help>
+        </properties>
+        <children>
+          <node name="cpu">
+            <properties>
+              <help>Show CPU info</help>
+            </properties>
+            <command>lscpu</command>
+            <children>
+              <node name="detail">
+                <properties>
+                  <help> Show system CPU details</help>
+                </properties>
+                <command>less $_vyatta_less_options --prompt='.cpu info' -- /proc/cpuinfo</command>
+              </node>
+              <node name="summary">
+                <properties>
+                  <help>Show CPU's on system</help>
+                </properties>
+                <command>${vyatta_bindir}/vyatta-cpu-summary.pl</command>
+              </node>
+            </children>
+          </node>
+
+          <node name="dmi">
+            <properties>
+              <help>Show system DMI details</help>
+            </properties>
+            <command>${vyatta_bindir}/vyatta-show-dmi</command>
```

Comment:
Any reason we are not calling dmidecode?
Anyway, the script is small, I suppose it can be renamed to dmi.sh and moved to the op mode scripts in vyos-1x

Reply:
as far as i see dmidecode is a more extensive output than the data fetched from the original vyatta-show-dmi script. 

Another question is if this information should be placed anywhere else instead of show hardware dmi ?